extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -66,327 +66,364 @@ export const RTC_ERROR_ENUM = {
   MACRO_EXPAND_TIMEOUT: '11',
 };
 
-/**
- * @param {string} error
- * @param {string} callout
- * @param {!Window} win
- * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
- * @param {number=} opt_rtcTime
- * @return {!Promise<!rtcResponseDef>}
- * @private
- */
-function buildErrorResponse_(
-  error, callout, errorReportingUrl, win, ampDoc, opt_rtcTime) {
-  dev().warn(TAG, `RTC callout to ${callout} caused ${error}`);
-  if (errorReportingUrl) {
-    sendErrorMessage(error, errorReportingUrl, win, ampDoc);
+export class RealTimeConfigManager {
+  /**
+   * @param {!./amp-a4a.AmpA4A} a4aElement
+   */
+  constructor(a4aElement) {
+    /** @private {!./amp-a4a.AmpA4A} */
+    this.a4aElement_ = a4aElement;
+
+    /** @private {!Object} */
+    this.urlReplacements_ = Services.urlReplacementsForDoc(
```

Comment:
Maybe it doesn't matter but why not lazily load this?  If the pub never includes rtc-config then this is called unnecessarily